data/countries/de/he/wetterau/data.yaml

```
@@ -86,7 +86,7 @@ state: Hessen
 type: REGIONAL_CHAPTER
 urls:
   - type: WEBSITE
-    url: http://gruene-karben.de/
+    url: http://gruene-karben.de
```

```suggestion
    url: http://gruene-karben.de/
```

Technically the URL must have at least the root path IMO, otherwise it will be a redirect again.